VTEX - Catalog API.json

```diff
@@ -4802,6 +3869,161 @@
+                                        },
+                                        "CubicWeight": {
+                                            "type": "number",
+                                            "description": "[Cubic weight](https://help.vtex.com/en/tutorial/understanding-the-cubic-weight-factor--tutorials_128)."
+                                        },
+                                        "IsKit": {
+                                            "type": "boolean",
+                                            "description": "Flag to set whether the product SKU is made up of one or more SKUs, thereby becoming a bundle. Must be enabled if you are adding a bundle. Once activated, the flag cannot be reverted."
+                                        },
+                                        "CreationDate": {
+                                            "type": "string",
+                                            "description": "Date and time of the SKU's creation."
+                                        },
+                                        "RewardValue": {
+                                            "type": "number",
+                                            "description": "Credit that the customer receives when finalizing an order of one specific SKU unit. By filling this field out with `1`, the customer gets U$ 1 credit on the site."
+                                        },
+                                        "EstimatedDateArrival": {
+                                            "type": "string",
+                                            "description": "SKU estimated arrival date in [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) format, when the product is on pre-sale. You must take into consideration both the launch date and the freight calculation for the arrival date.",
+                                            "nullable": true
+                                        },
+                                        "ManufacturerCode": {
+                                            "type": "string",
+                                            "description": "Provided by the manufacturers to identify their product. This field should be filled in if the product has a specific manufacturer’s code."
+                                        },
+                                        "CommercialConditionId": {
+                                            "type": "integer",
+                                            "description": "Used to define SKU specific promotions or installment rules. In case of no specific condition, use `1` (default value). This field does not accept `0`. Find out more by reading [Registering a commercial condition](https://help.vtex.com/tutorial/registering-a-commercial-condition--tutorials_445)."
```

Comment:
```suggestion
                                            "description": "Commercial condition ID, used to define SKU specific promotions or installment rules. In case of no specific condition, use `1` (default value). This field does not accept `0`. Learn more at [Registering a commercial condition](https://help.vtex.com/tutorial/registering-a-commercial-condition--tutorials_445).",
                                            "default": 1
```